### ISSUE TITLE
ExplorerPatcher Crashes on Windows 11 (Version 22h2) (OS Build 22621.2283)

### ISSUE THREAD

**Issue by JoBanJo** [valinet/ExplorerPatcher#2081]
After updating to the latest version of Windows 11 (Version 22h2) (OS Build 22621.2283), When clicking the windows icon on the task bar or press the windows key on my keyboard nothing happens

Steps to Reproduce:

Install Windows 11 (Version 22h2) (OS Build 22621.2283).
Install the latest version of ExplorerPatcher.

**Comment by redleafnew** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271

Does this work?

**Comment by redleafnew** [comment]
> See [#1702 (comment)](https://github.com/valinet/ExplorerPatcher/discussions/1702#discussioncomment-6580555) for a likely fix - worked for me!

***Huge thanks to @IMMX5 for sharing this tip!***... have to admit, as it was posted a few weeks ago I wasn't sure if it would work for `KB5029263` (which came out literally just yesterday) but very glad to confirm that it does! 🎉

***

## ✅ Fix Summary
I realise it isn't a huge time-saving, but in an effort to help as much as I can, I have put a summary of the steps to get EP working again below...
1. Uninstall EP
2. Remove these registry keys (you can save the below as a .reg file if you prefer)...
```
Windows Registry Editor Version 5.00

[-HKEY_CURRENT_USER\Software\ExplorerPatcher]
[-HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
```
3. Reboot
4. Reinstall EP (the latest / same version you that was installed previously will work)

NB: The above should work up to and including `Windows 11 22H2 build 22621.2134`

***

Also, I want to thank everyone for their efforts on this and so thanks also to @AgentFire for your tip...
> Press Win+R to quickly crash the patched explorer (to induce failsafe mode).

I had EP installed on another machine and found pressing `Win + R` repeatedly caused EP to self-deactive ***MUCH*** more quickly, so thanks for sharing this too! 👍🏼

_Originally posted by @gaiking-uk in https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271_

**Comment by JoBanJo** [comment]
> [#1802 (comment)](https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271)
> 
> Does this work?

It didn't work for me unfortunately

**Comment by redleafnew** [comment]
Does renaming the HKCU\Software\ExplorerPatcher\twinui.pcshell key to something else work?

**Comment by redleafnew** [comment]
And 
https://github.com/valinet/ExplorerPatcher/issues/1828#issuecomment-1675937607

**Comment by JoBanJo** [comment]
> Does renaming the HKCU\Software\ExplorerPatcher\twinui.pcshell key to something else work?

The task bar seems to be working now but clicking the windows icon or search icon does nothing.

**Comment by JoBanJo** [comment]
> Does renaming the HKCU\Software\ExplorerPatcher\twinui.pcshell key to something else work?

> And
https://github.com/valinet/ExplorerPatcher/issues/1828#issuecomment-1675937607

No

**Comment by evandrosouza89** [comment]
> ```
>           > See [#1702 (comment)](https://github.com/valinet/ExplorerPatcher/discussions/1702#discussioncomment-6580555) for a likely fix - worked for me!
> ```
> 
> _**Huge thanks to @IMMX5 for sharing this tip!**_... have to admit, as it was posted a few weeks ago I wasn't sure if it would work for `KB5029263` (which came out literally just yesterday) but very glad to confirm that it does! 🎉
> 
> ## ✅ Fix Summary
> I realise it isn't a huge time-saving, but in an effort to help as much as I can, I have put a summary of the steps to get EP working again below...
> 
> 1. Uninstall EP
> 2. Remove these registry keys (you can save the below as a .reg file if you prefer)...
> 
> ```
> Windows Registry Editor Version 5.00
> 
> [-HKEY_CURRENT_USER\Software\ExplorerPatcher]
> [-HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
> ```
> 
> 3. Reboot
> 4. Reinstall EP (the latest / same version you that was installed previously will work)
> 
> NB: The above should work up to and including `Windows 11 22H2 build 22621.2134`
> 
> Also, I want to thank everyone for their efforts on this and so thanks also to @AgentFire for your tip...
> 
> > Press Win+R to quickly crash the patched explorer (to induce failsafe mode).
> 
> I had EP installed on another machine and found pressing `Win + R` repeatedly caused EP to self-deactive _**MUCH**_ more quickly, so thanks for sharing this too! 👍🏼
> 
> _Originally posted by @gaiking-uk in [#1802 (comment)](https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271)_

it worked for me thank you

**Comment by JoBanJo** [comment]
> it worked for me thank you

What version of windows and what os build are you on?

**Comment by HaneruRenjyo** [comment]
> ```
>           > See [#1702 (comment)](https://github.com/valinet/ExplorerPatcher/discussions/1702#discussioncomment-6580555) for a likely fix - worked for me!
> ```
> 
> _**Huge thanks to @IMMX5 for sharing this tip!**_... have to admit, as it was posted a few weeks ago I wasn't sure if it would work for `KB5029263` (which came out literally just yesterday) but very glad to confirm that it does! 🎉
> 
> ## ✅ Fix Summary
> I realise it isn't a huge time-saving, but in an effort to help as much as I can, I have put a summary of the steps to get EP working again below...
> 
> 1. Uninstall EP
> 2. Remove these registry keys (you can save the below as a .reg file if you prefer)...
> 
> ```
> Windows Registry Editor Version 5.00
> 
> [-HKEY_CURRENT_USER\Software\ExplorerPatcher]
> [-HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
> ```
> 
> 3. Reboot
> 4. Reinstall EP (the latest / same version you that was installed previously will work)
> 
> NB: The above should work up to and including `Windows 11 22H2 build 22621.2134`
> 
> Also, I want to thank everyone for their efforts on this and so thanks also to @AgentFire for your tip...
> 
> > Press Win+R to quickly crash the patched explorer (to induce failsafe mode).
> 
> I had EP installed on another machine and found pressing `Win + R` repeatedly caused EP to self-deactive _**MUCH**_ more quickly, so thanks for sharing this too! 👍🏼
> 
> _Originally posted by @gaiking-uk in [#1802 (comment)](https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271)_

EP Crashing on [Windows 11 (Version 22h2) (OS Build 22621.2283)] fixed, thanks a lot

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2051

Please close this one.

Possible fixes:

1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**

**Comment by JoBanJo** [comment]
> 1. In ExplorerPatcher Properties. 

I can't access ExplorerPatchers properties as my task bar doesn't currently work due to ExplorerPatcher crashing constantly.

**Comment by JoBanJo** [comment]
> duplicate of: #2051
> 
> Please close this one.
> 
> Possible fixes:
> 
> 1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
> 2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
> 3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**

Unfortunately this did not fix the problem for me. When clicking the windows icon on the task bar or press the windows key on my keyboard nothing happens.

**Comment by pyrates999** [comment]
> > duplicate of: #2051
> > Please close this one.
> > Possible fixes:
> > 
> > 1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
> > 2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
> > 3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**
> 
> Unfortunately this did not fix the problem for me. When clicking the windows icon on the task bar or press the windows key on my keyboard nothing happens.

that's not a crash from what you described.

**Comment by iuskov-nablet** [comment]
> > duplicate of: #2051
> > Please close this one.
> > Possible fixes:
> > 
> > 1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
> > 2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
> > 3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**
> 
> Unfortunately this did not fix the problem for me. When clicking the windows icon on the task bar or press the windows key on my keyboard nothing happens.

I'm observing absolute same behavior. The issues began since  KB5030219 update several days ago. Windows build is 22621.2283.
Initially I had explorer crashes, now after patcher has re-installed the explorer just do nothing when I click on taskbar's windows button.

**Comment by JoBanJo** [comment]
> I'm observing absolute same behavior. The issues began since KB5030219 update several days ago. Windows build is 22621.2283. Initially I had explorer crashes, now after patcher has re-installed the explorer just do nothing when I click on taskbar's windows button.

This is exactly what I'm getting.

**Comment by RepentAndBelieveTheGospel** [comment]
I unistalled EP last week after updating Windows, because explorer kept crashing. Today I removed the registry keys, restarted, installed EP again and it took awhile before the taskbar icons appeared, but Start Menu wasn't working. I restarted explorer from the Task Manager (it's under Windows Explorer) and now it works! Thank you.

**Comment by JoBanJo** [comment]
> now it works! Thank you.

What version / os build are you on?

**Comment by iuskov-nablet** [comment]
> I restarted explorer from the Task Manager (it's under Windows Explorer) and now it works! Thank you.
After second PE  re-install with registry cleaning my patcher also working now. Thank for the info!

**Comment by pyrates999** [comment]
We need to find out what registry change was causing EP properties application to crash.

**Comment by RepentAndBelieveTheGospel** [comment]
> What version / os build are you on?

The one from the title (22H2 22621.2283).

**Comment by JoBanJo** [comment]
> We need to find out what registry change was causing EP properties application to crash.

The issue is you can't use the windows key or icon and sometimes the search button doesn't work. Some people have fixed it by deleting the registry entries, restarting. then reinstalling EP. However this has not worked for me or [iuskov-nablet](https://github.com/iuskov-nablet)

**Comment by iuskov-nablet** [comment]
> We need to find out what registry change was causing EP properties application to crash.

I believe the reason is not registry but explorer binaries changes after windows update. I  have noticed that EP re-loads debug symbols after re-installation. Possible the reason if issue is using old obsolete symbols for updated binaries but this is only my guess.

**Comment by Kryomancer** [comment]
The only 100% sure solution is to wait for EP to be updated to match the latest Windows version. As we can see from the releases, As we can see from the releases, latest EP is for Windows build 22621.1992, and any later than that are likely to run into issues until there's an update.

**Comment by pyrates999** [comment]
> > We need to find out what registry change was causing EP properties application to crash.
> 
> I believe the reason is not registry but explorer binaries changes after windows update. I have noticed that EP re-loads debug symbols after re-installation. Possible the reason if issue is using old obsolete symbols for updated binaries but this is only my guess.

EP will download the symbols for the version of windows you have installed.  It will do this during the install and after a new update has been installed.

**Comment by Kryomancer** [comment]
> > > We need to find out what registry change was causing EP properties application to crash.
> > 
> > 
> > I believe the reason is not registry but explorer binaries changes after windows update. I have noticed that EP re-loads debug symbols after re-installation. Possible the reason if issue is using old obsolete symbols for updated binaries but this is only my guess.
> 
> EP will download the symbols for the version of windows you have installed. It will do this during the install and after a new update has been installed.

The problem is the symbols have not yet been updated for the latest build of Windows so it will invariably crash trying to use the old ones.

**Comment by ApuchYz** [comment]
Worked for me:
1. Run CMD as Admin
2. paste: "wusa.exe /uninstall /kb:5030219"

It will uninstall the update that caused the problem.

**Comment by Bo-Liu-BL** [comment]
> ```
>           > See [#1702 (comment)](https://github.com/valinet/ExplorerPatcher/discussions/1702#discussioncomment-6580555) for a likely fix - worked for me!
> ```
> 
> _**Huge thanks to @IMMX5 for sharing this tip!**_... have to admit, as it was posted a few weeks ago I wasn't sure if it would work for `KB5029263` (which came out literally just yesterday) but very glad to confirm that it does! 🎉
> ## ✅ Fix Summary
> 
> I realise it isn't a huge time-saving, but in an effort to help as much as I can, I have put a summary of the steps to get EP working again below...
> 
>     1. Uninstall EP
> 
>     2. Remove these registry keys (you can save the below as a .reg file if you prefer)...
> 
> 
> ```
> Windows Registry Editor Version 5.00
> 
> [-HKEY_CURRENT_USER\Software\ExplorerPatcher]
> [-HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher]
> ```
> 
>     3. Reboot
> 
>     4. Reinstall EP (the latest / same version you that was installed previously will work)
> 
> 
> NB: The above should work up to and including `Windows 11 22H2 build 22621.2134`
> 
> Also, I want to thank everyone for their efforts on this and so thanks also to @AgentFire for your tip...
> 
> > Press Win+R to quickly crash the patched explorer (to induce failsafe mode).
> 
> I had EP installed on another machine and found pressing `Win + R` repeatedly caused EP to self-deactive _**MUCH**_ more quickly, so thanks for sharing this too! 👍🏼
> 
> _Originally posted by @gaiking-uk in [#1802 (comment)](https://github.com/valinet/ExplorerPatcher/issues/1802#issuecomment-1670989271)_

It worked for me, too.

**Comment by evandrosouza89** [comment]
> > it worked for me thank you
> 
> What version of windows and what os build are you on?

22H2 22621.2283

**Comment by Amrsatrio** [comment]
Please install the newly released EP build. The new version should never do irrecoverable crashes like these anymore. There should be no more major issues on 22621 builds up to 22621.2359.

**Comment by BMJ-pdx** [comment]
The fubarosity seems to be resolved, with no tricks required, by 22621.2283.57.2 .
(This is on Windoze 10 Home 22H2 19045.3448 .)
